src/App/Fossa/Analyze.hs

```diff
@@ -180,10 +181,14 @@ runDependencyAnalysis ::
   ) =>
   -- | Analysis base directory
   Path Abs Dir ->
+  -- | Filters
   AllFilters ->
+  -- | An optional path prefix to prepend to paths of discovered manifestFiles
+  Maybe (Path Rel Dir) ->
```

Comment:
[non-blocking] Please strongly consider making a newtype for this and sticking it next to the `ancestry` methods. Maybe `FileAncestor`?

Then you can take `Maybe FileAncestor`, which more clearly denotes what exactly this argument is for.

(Yes, I wish we'd have done this with base directory and many other special variables we just pass around to everything, and intend to one day refactor to do so).

Reply:
That's a good idea. Done!